tests/disk_management/test_disk_read.py

```diff
@@ -0,0 +1,49 @@
+import subprocess
+import unittest
+import os
+import egg.disk_management
+
+from tests.disk_management import load_expected_results, VHD_DIR
+
+
+@unittest.skipUnless(os.getuid() == 0, None)
+class TestDiskRead(unittest.TestCase):
+
+    scenario_data: dict
+    vhd_name: str
+    vhd_table: str
+    loop_path: str
+
+    @classmethod
+    def fillClassVariables(cls, scenario_data):
+        cls.vhd_name = scenario_data.get('vhd_name')
+        cls.vhd_table = VHD_DIR + cls.vhd_name + '/vhd.table'
+        os.system(VHD_DIR + cls.vhd_name + '/setup.sh ' + cls.vhd_table)
+        cls.expected = load_expected_results(cls.vhd_name)
+        cls.loop_path = subprocess.check_output('cat ./loop_path.tmp', shell=True).rstrip().decode("utf-8")
+
+    def test_disk(self) -> None:
```

Comment:
You don't need to type hint functions that don't return anything. Remember than type hints are just hints, they are not mandatory!

```suggestion
    def test_disk(self):
```